insights/client/connection.py

```diff
@@ -616,92 +661,87 @@ def do_group(self):
     def _legacy_api_registration_check(self):
         '''
         Check registration status through API
+
+        Returns:
+            unreg_date if unregistered
```

Comment:
Where did this _unreg_date_ come from? In the response, it’s called _unregistered_at_, in the code, it’s _unreg_status_.